src/documents/CoreAPI.md

```diff
@@ -567,13 +567,13 @@ accepted from external users.
 
 ### fluid.log([logLevel, arg1, ... argn])
 
-* `logLevel {LogLevel}` [optional] One of the members of the `evel` structure or in general some `fluid.marker` with a numeric value holding a priority
+* `logLevel {LogLevel}` [optional] One of the members of the `fluid.logLevel` structure or in general some `fluid.marker` with a numeric value holding a priority
 * `arg1 ... argn {Stringable}` A sequence of arguments to be logged. By default these will be handed unchanged to the multiple arguments of `console.log`. It is wise to convert these
   to `String` yourself (e.g. by `JSON.stringify`) if i) this is safe, and ii) you would not be satisfied by the environment's rendering.
 
 Log a message to a suitable environmental console (in current implementations, `console.log`). If the first argument to `fluid.log` is
-one of the members of the [`evel`](#fluidloglevel) structure, this will be taken as the priority of the logged message - else it
-will default to [`evel.INFO`](#fluidloglevel). If the logged message priority does not exceed that set by the most recent call to the [`fluid.setLogging`](#fluidsetloggingloglevel) function,
+one of the members of the [`fluid.logLevel`](#fluidloglevel) structure, this will be taken as the priority of the logged message - else it
+will default to [`fluid.logLevel.INFO`](#fluidloglevel). If the logged message priority does not exceed that set by the most recent call to the [`fluid.setLogging`](#fluidsetloggingloglevel) function,
```

Comment:
Not your change but we may as well fix it: `#fluidloglevel` should be `#fluid-loglevel`.

Reply:
So, we need to regenerate the live site using this or the results of the previous pull.

That link is absolutely correct if you view the page on GitHub:

[https://github.com/fluid-project/infusion-docs/blob/master/src/documents/CoreAPI.md#fluidloglevel](https://github.com/fluid-project/infusion-docs/blob/master/src/documents/CoreAPI.md#fluidloglevel)

It will be correct on the site once we regenerate it using the results of this PR.

FYI, "npm test" now checks every internal and external link, I did find one broken link using that and will fix that shortly.

Reply:
The other broken link is now fixed.

Reply:
I also confirmed that the link works when opening the Docpad output locally.  @simonbates, just to confirm, how did you know this was broken?  Just knew it by heart or compared with the live site?  I want to make extra sure there isn't a difference between Docpad versions or platforms or anything odd.

Reply:
Sorry @the-t-in-rtf you are totally right.

I generated the docs locally using docpad on my computer at home and that's where I saw the `#fluid-loglevel` but now I'm in work and I can't reproduce. I tried on both Linux and Windows (in the remote chance that that would make a difference -- at home I was on Windows) but I'm seeing `#fluidloglevel` consistently.

I'm guessing I did something dumb like generate the docs without updating the node dependencies.

Reply:
Great, do please let me know.  It may be that we need to add a note to the README if it's a version problem.

Reply:
Ok, I've done a bit of investigation. It looks like I had run `npm update` but not cleaned out the `node_modules` beforehand. I had the latest dependencies, but also modules hanging around from before. In particular `docpad-plugin-marked` and friends.

I made a copy and ran some experiments.

Running with fresh `node_modules` I get the expected output (`#fluidloglevel`):

```
> docpad run --env static
info: Welcome to DocPad v6.79.4 (global installation: npm_global\node_modules\docpad)
notice: Please donate to DocPad or have your company sponsor it: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins: ghpages, handlebars, highlightjs, markit
info: Environment: static
info: Server started on http://0.0.0.0:9778
info: In your web browser use http://127.0.0.1:9778
info: Generating...
info: Generated 257/258 files in 15.908 seconds
info: Watching setup starting...
info: Watching setup
info: The action completed successfully
```

Leaving my existing `node_modules` in place and after running `npm install`:

```
> docpad run --env static
info: Welcome to DocPad v6.79.4 (global installation: npm_global\node_modules\docpad)
notice: Please donate to DocPad or have your company sponsor it: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins: ghpages, handlebars, highlightjs, marked, markit
info: Environment: static
info: Server started on http://0.0.0.0:9778
info: In your web browser use http://127.0.0.1:9778
info: Generating...
info: Generated 257/258 files in 16.079 seconds
info: Watching setup starting...
info: Watching setup
info: The action completed successfully
```

Note the presence of `marked` in the Plugins list. And it's listed before `markit` suggesting that it might be given precedence over markit. In this case, I get the old output (`#fluid-loglevel`).

Now, if I remove `docpad-plugin-marked` and run `docpad`, I get the desired output (`#fluidloglevel`):

```
> rmdir -Force -Recurse .\node_modules\docpad-plugin-marked\
> docpad run --env static
info: Welcome to DocPad v6.79.4 (global installation: npm_global\node_modules\docpad)
notice: Please donate to DocPad or have your company sponsor it: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins: ghpages, handlebars, highlightjs, markit
info: Environment: static
info: Server started on http://0.0.0.0:9778
info: In your web browser use http://127.0.0.1:9778
info: Generating...
info: Generated 257/258 files in 16.582 seconds
info: Watching setup starting...
info: Watching setup
info: The action completed successfully
```

It looks like it's necessary to remove a local `docpad-plugin-marked` from `node_modules` when we switch to markit.

On my other computers, I believe that I had removed the `node_modules` and refreshed the dependencies before looking at these changes. But on my computer at home, I had not done the step of cleaning out the `node_modules`.

Reply:
So, if we released versions of this project independently, I would argue something like that is a good candidate to put in the release notes for a particular version (i.e. the version where markdown-it is introduced).  Currently, we seem to only have versions that correspond to the associated Infusion release.  It seems a bit much to put something in the README, you could argue that clearing node_modules is pretty much a universal requirement for almost all of our work.  I can comment on the mailing list about the change, would that be enough, @simonbates?

Reply:
What do you think about putting a general note on DocPad plugins into the README?

Something at the end under a heading "A Note on DocPad plugins":

"Over time, we will occasionally change the DocPad plugins that we use in this project (such as replacing the Markdown processor used). When this happens, it will be necessary to clean out the previous plugins so that DocPad does not use them. For example, by removing the `node_modules` directory and running `npm install`."

Reply:
I added a slightly harder warning to the README.  Please comment there.